attestation-service/Cargo.toml

```diff
@@ -39,7 +39,7 @@ scroll = { version = "0.11.0", default-features = false, features = ["derive"],
 serde.workspace = true
 serde_json.workspace = true
 serde_variant = "0.1.2"
-sev = { version = "= 1.1.0", features = ["openssl"], optional = true }
+sev = { version = "= 1.2.0", features = ["openssl", "snp"], optional = true }
```

Comment:
Do we still need to set an exact version with these changes?

Reply:
no we don't, i've pushed this change